src/OrbitGl/BatcherTest.cpp

```diff
@@ -23,6 +23,27 @@ class FakeBatcher : public OpenGlBatcher {
   explicit FakeBatcher(BatcherId id, PickingManager* picking_manager = nullptr)
       : OpenGlBatcher(id, picking_manager) {}
 
+  // Auxiliary methods to simplify the addition of lines, boxes and triangles.
+  using Batcher::AddLine;
+  void AddLine(Vec2 from, Vec2 to, float z, const Color& color,
+               std::unique_ptr<PickingUserData> user_data = nullptr) {
+    Color picking_color = PickingId::ToColor(PickingType::kLine, GetNumElements(), GetBatcherId());
+    return AddLine(from, to, z, color, picking_color, std::move(user_data));
+  }
+  using Batcher::AddBox;
+  void AddBox(const Box& box, const Color& color,
+              std::unique_ptr<PickingUserData> user_data = nullptr) {
+    Color picking_color = PickingId::ToColor(PickingType::kBox, GetNumElements(), GetBatcherId());
+    return AddBox(box, {color, color, color, color}, picking_color, std::move(user_data));
+  }
+  using Batcher::AddTriangle;
+  void AddTriangle(const Triangle& triangle, const Color& color,
+                   std::unique_ptr<PickingUserData> user_data = nullptr) {
+    Color picking_color =
+        PickingId::ToColor(PickingType::kTriangle, GetNumElements(), GetBatcherId());
+    return AddTriangle(triangle, {color, color, color}, picking_color, std::move(user_data));
+  }
+
```

Comment:
These shadow a function from the base class. That's not good practice. Couldn't you use a free function instead?

Reply:
I renamed to an internal method Add****Helper, what do you think?